.circleci/config.yml

```diff
@@ -4,6 +4,11 @@ orbs:
   buildevents: honeycombio/buildevents@0.2.3
 
 executors:
+  docker-publisher:
+    environment:
+      IMAGE_NAME: honeycombio/honeycomb-opentracing-proxy
+    docker:
+      - image: circleci/buildpack-deps:stretch
```

Comment:
is it required to specify stretch? Unless it only works there, I would suggest either using the current stable (buster) or omitting it entirely in hopes that that will use whatever's current.

Reply:
I'm not sure if it's required, I've removed it though and we can test this out shortly when we do the major version bump